src/SymbolManager.cpp

```diff
@@ -108,17 +109,49 @@ void SymbolManager::refresh()
  * File list support functions
  */
 
+enum TableColumns {
+	FILENAME = 0,
+	DESTINATION_SLOT,
+	DESTINATION_SEGMENT,
+	LAST_REFRESH,
+};
+
 void SymbolManager::initFileList()
 {
 	treeFiles->clear();
 	for (int i = 0; i < symTable.symbolFilesSize(); ++i) {
 		auto* item = new QTreeWidgetItem(treeFiles);
-		item->setText(0, symTable.symbolFile(i));
+		item->setText(FILENAME, symTable.symbolFile(i));
 		auto fmt = QLocale().dateTimeFormat(QLocale::ShortFormat);
-		item->setText(1, symTable.symbolFileRefresh(i).toString(fmt));
+		createComboBox(i);
+		item->setText(LAST_REFRESH, symTable.symbolFileRefresh(i).toString(fmt));
 	}
 }
 
+static const char* destinationLabels[] = {
+	"All",
+	"0-0", "0-1", "0-2", "0-3",
+	"1-0", "1-1", "1-2", "1-3",
+	"2-0", "2-1", "2-2", "2-3",
+	"3-0", "3-1", "3-2", "3-3",
+};
+
+void SymbolManager::createComboBox(int row)
+{
+	auto* combo = new QComboBox();
+	combo->setEditable(false);
+	combo->insertItems(0, QStringList(std::begin(destinationLabels), std::end(destinationLabels)));
+	combo->setCurrentIndex(0);
```

Comment:
Do I understand correctly that initially the combo box will always show "All"?
I would expect (here or somewhere else in the patch) to see code that looks at the currently allowed slots of the symbol, and then set the initial value of the combo box accordingly.

Reply:
Most symbol files don't store information of allowed slots, so that depends entirely on the MSX configuration. Besides the debugger already allows you to set any slot you want, not just currently allowed slots:
![image](https://user-images.githubusercontent.com/7815819/214306347-b0f8eea1-4b9e-42e9-8cf2-dc914a3fda8d.png)